apps/src/templates/VersionHistory.jsx

```diff
@@ -1,12 +1,18 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import VersionRow from './VersionRow';
+
+// Disabling import order due to circular dependencies errors from build.
```

Comment:
As above, I'd like to see some more details in this comment, and I'm curious why we're disabling the rule line-by-line rather than as a group.